lib/documentLoaders/node.js

```diff
@@ -83,7 +83,7 @@ module.exports = ({
     const statusText = http.STATUS_CODES[res.statusCode];
     if(res.statusCode >= 400) {
       throw new JsonLdError(
-        'URL could not be dereferenced: ' + statusText,
+        `URL "${url}" could not be dereferenced: ${statusText}`,
```

Comment:
This is a change from our usual style.  URL is already a param in the details so could be omitted in the message text.  Could also move the statusText to details.

Reply:
I'll merge PR, but may change this text later.